massgov/openmass
# DP-22689: Make About section for org page (second follow up)

**Description:**
Changed the About paragraph type label to About Elected Official.

**Jira:** (Skip unless you are MA staff)
https://massgov.atlassian.net/browse/DP-22689


**To Test:**
- Edit an Organization node.
- Click on the "Content" tab.
- Edit or create an Organization Section.
- Verify the button is "Add About Elected Official" instead of "Add About".

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
